packages/zoxide/build.sh

```diff
@@ -12,5 +13,6 @@ termux_step_post_make_install() {
 	install -Dm644 contrib/completions/zoxide.bash "$TERMUX_PREFIX"/share/bash-completion/completions/zoxide
 	install -Dm644 contrib/completions/_zoxide "$TERMUX_PREFIX"/share/zsh/site-functions/_zoxide
 	install -Dm644 contrib/completions/zoxide.fish "$TERMUX_PREFIX"/share/fish/vendor_completions.d/zoxide.fish
+	install -Dm644 contrib/completions/zoxide.nu "$TERMUX_PREFIX"/share/nushell/vendor/autoload/zoxide.nu
```

Comment:
While we're here already, can you please also add the Elvish completions?
https://github.com/ajeetdsouza/zoxide/blob/main/contrib/completions/zoxide.elv
```suggestion
	install -Dm644 contrib/completions/zoxide.nu "$TERMUX_PREFIX"/share/nushell/vendor/autoload/zoxide.nu
	install -Dm644 contrib/completions/zoxide.elv "${TERMUX_PREFIX}"/share/elvish/lib/zoxide.elv
```

If you want to use a more generic approach like `yazi` for example does that's also an option.
https://github.com/termux/termux-packages/blob/ad011b0dd46a74f9c42082eb86880f58622e74b3/packages/yazi/build.sh#L26-L30